sources/packages/api/src/route-controllers/user/user.controller.ts

```diff
@@ -31,7 +34,9 @@ export class UserController extends BaseController {
     @UserToken() userToken: IUserToken,
   ): Promise<UserSyncInfoDto> {
     try {
-      const syncedUser = await this.service.synchronizeUserInfo(userToken);
+      const syncedUser = await this.studentService.synchronizeUserInfo(
```

Comment:
This is syncedStudent and not syncedUser right? If yes, can we name it accordingly

Reply:
I struggled a bit with the name also because it is gonna update data on the user(first name, last name, and email) and also on the student (DOB and gender).
So, we are basically using User data to update Users/Students table.
Maybe we can say this.studentService.synchronizeFromUserInfo, what do you think?